### PR TITLE
doc(changelog): Add changelog for an orca change

### PR DESCRIPTION
**Pull request by kirangodishala**
Add change log for the PR: https://github.com/spinnaker/orca/pull/4781.

### CODE REVIEW

File: content/en/docs/releases/next-release-preview/index.md [hunk 1]
```diff
@@ -39,6 +39,31 @@ controller:
 #### Orca
 Updates Orca's SavePipelineTask to support bulk saves using the updated functionality in the front50 bulk save endpoint.
 
+Keys from the stage context's outputs section can now be removed (there by reducing the context size significantly).
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:42]: Please include a link to the PR here.

Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:42]: This feature only supports these specific tasks, right?

Comment (kirangodishala) [reply content/en/docs/releases/next-release-preview/index.md]: That is correct.

Comment (kirangodishala) [reply content/en/docs/releases/next-release-preview/index.md]: Done

File: content/en/docs/releases/next-release-preview/index.md [hunk 2]
```diff
@@ -39,6 +39,31 @@ controller:
 #### Orca
 Updates Orca's SavePipelineTask to support bulk saves using the updated functionality in the front50 bulk save endpoint.
 
+With https://github.com/spinnaker/orca/pull/4781, keys from the stage context's outputs section can now be removed (there by reducing the context size significantly).
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:42]:
```suggestion
With https://github.com/spinnaker/orca/pull/4781, keys from the stage context's outputs section can now be removed (there by reducing the context size significantly).   There's currently only support for these tasks:
* PromoteManifestKatoOutputsTask
* WaitOnJobCompletionTask
* ResolveDeploySourceManifestTask
* BindProducedArtifactsTask
```
or something like this.

**Review by dbyron-sf — APPROVED** [review]

**Pull request merged**